models/fates/R/write.configs.FATES.R

```diff
@@ -19,24 +19,40 @@
 ##' @return none
 ##' @export
 ##' @author Mike Dietze, Shawn Serbin
-##-------------------------------------------------------------------------------------------------#
+##-------------------------------------------------------------------------------------------------# # nolint
+
+# example trait.values list
+trait.values <- list()
+trait.values$temperate.coniferous <- list()
+trait.values$env <- list()
+
+tc_traits <- as.data.frame(matrix(c(9999, 8888), nrow=1, ncol=2))
+colnames(tc_traits) <- c("leaf_turnover_rate", "veg_respiration_Q10")
+# Fates names for these params are "fates_leaf_long", "q10_mr"
+
+trait.values$temperate.coniferous <- tc_traits
+settings <- '/pecan/models/fates/R/test.fates.xml'
+run.id <- 'test_oneyear'
+
+# example trait.values list
+
 write.config.FATES <- function(defaults, trait.values, settings, run.id){
 
    ## site information
    site <- settings$run$site
-   site.id <- as.numeric(site$id)
+   site.id <- site$id # change1: 772 -> SOD1
```

Comment:
not following the logic of the change here. Comment is also unclear to a general user